test/System/CGroup/TypesSpec.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE CPP #-}
```

Comment:
I'd really like to avoid doing anything with CPP if possible.  Maybe we could use a wrapper around `it` which does nothing on windows.